Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -568,7 +573,20 @@ public override void StartDriver()
                         EdgeDriverService EDService = EdgeDriverService.CreateDefaultService();//CreateDefaultServiceFromOptions(EDOpts);
                         EDService.HideCommandPromptWindow = HideConsoleWindow;
                         Driver = new EdgeDriver(EDService, EDOpts, TimeSpan.FromSeconds(Convert.ToInt32(HttpServerTimeOut)));
-
+                        // Changes to be done here or in the Edge mode
+                        if(OpenInIEMode == true)
+                        {
+                            var ieOptions = new InternetExplorerOptions();
+                            ieOptions.AttachToEdgeChrome = true;
+                            //change the path accordingly
+                            ieOptions.EdgeExecutablePath = UserProfileFolderPath;
+
+                            var driver = new InternetExplorerDriver(ieOptions);
```

Comment:
please use driver from line 575,only attach to edge chrome should be in if condition